src/navigator/types.tsx

```diff
@@ -267,6 +267,10 @@ export type StackParamList = {
       }
     | undefined
   [Screens.SwapScreenWithBack]: { fromTokenId: string } | undefined
+  [Screens.TabNavigator]: {
+    initialScreen?: Screens
+    fromModal?: boolean
```

Comment:
is this param required?

Reply:
It's needed for `navigateHome` in `src/navigator/NavigationService.ts`.